Backend/UserService.cs

```diff
@@ -119,5 +119,32 @@ public async Task<IdentityResult> UpdateUserAsync(User userToUpdate)
         {
             return await _userManager.UpdateAsync(userToUpdate);
         }
+
+        public UserInformations GetUserFromLDAP(string login, string password)
+        {
+            login += "@fri.uniza.sk";
+
+            OptionsLdap options = new OptionsLdap
+            {
+                SecureSocketLayer = false,
+                BaseDN = "DC=fri,DC=uniza,DC=sk",
+                HostName = "ldapbdc.fri.uniza.sk",
+                Port = 389,
+            };
+
+            AuthenticatorLdap authenticatorLdap = new AuthenticatorLdap(options);
+            UserInformations informations = authenticatorLdap.Authenticate(login, password);
+            return informations;
+        }
+
+        public async Task<bool> AddLdapUser(UserInformations informations, string password)
+        {
+            string[] names = informations.Name.Split(" ");
+            User user = new User(informations.Email, names[0], names[1]);
+            user.EmailConfirmed = true;
+            user.IsLdapUser = true;
+            var addResult = await AddUserAsync(user,password);
```

Comment:
Do not store the password, if not used (no send as parameter, it should not contain parameter for it

Reply:
Ok, will probably fix this but password which will be sent is "Heslo123" which is set up in controller. This password is only for cosmetics because addUserAsync checks passwords length, cases and numbers.

Reply:
It is just the hot fix, it should be solved better